docs/src/main/sphinx/admin/schedule-weight-example.json

```diff
@@ -0,0 +1,10 @@
+{
+  {
+    "name": "pipeline",
+    "schedulingWeight": 70,
+  },
+  {
+    "name": "adhoc",
+    "schedulingWeight": 30
+  }
+}
```

Comment:
empty line at end of file

Reply:
There is an empty line showing on VSCode but not in the PR...